packages/xo-server-backup-reports/src/index.js

```diff
@@ -365,9 +366,10 @@ class BackupReportsXoPlugin {
     })
   }
 
-  async _ngVmHandler(log, { name: jobName }, schedule, force) {
+  async _ngVmHandler(log, { name: jobName, settings }, schedule, force) {
     const xo = this._xo
 
+    const mailReceivers = get(() => settings[''].reportReceivers)
```

Comment:
`emailRecipients`?

Reply:
I'm not against using `emailRecipients` instead of  `mailReceivers` but we already use the second term and IMHO  we should probably use one term to not lead confusions.